lib/models/disk.rb

```diff
@@ -48,7 +49,7 @@ def api_format
     {
       "id": remote_id,
       "name": name,
-      "maximum_size_bytes": size ? size : 0, # Default to 0 if nil, otherwise API throws error
+      "storage_bytes": size ? size : 0, # Default to 0 if nil, otherwise API throws error
```

Comment:
Just to know, is there any reason behind this change? I assume that it's an on-prem-api change, but just to confirm.